ideascube/mediacenter/tests/test_remove_medias.py

```diff
@@ -0,0 +1,100 @@
+import os, stat
+
+from django.core.management import call_command
+
+import pytest
+
+from ideascube.mediacenter.models import Document
+
+pytestmark = pytest.mark.django_db
+CSV_PATH = os.path.join(os.path.dirname(__file__), 'data/metadata.csv')
+
+
+def write_metadata(metadata):
+    with open(CSV_PATH, 'w') as f:
+        f.write(metadata)
+
+
+def teardown_function(function):
+    if os.path.exists(CSV_PATH):
+        os.remove(CSV_PATH)
+
+
+def test_clean_orphans_should_not_fail_if_no_media_or_file():
+    call_command('remove_medias', 'clean_orphans')
+
+
+def test_clean_orphans_should_not_remove_files_associated_with_documents(settings):
+    metadata = ('title,summary,credits,path,preview\n'
+                'my video,my video summary,BSF,a-video.mp4,an-image.jpg\n')
+    write_metadata(metadata)
+    call_command('import_medias', CSV_PATH)
+    original_path = os.path.join(settings.MEDIA_ROOT, 'mediacenter/document', 'a-video.mp4')
+    preview_path = os.path.join(settings.MEDIA_ROOT, 'mediacenter/preview', 'an-image.jpg')
+    assert os.path.exists(original_path)
+    assert os.path.exists(preview_path)
+    call_command('remove_medias', 'clean_orphans')
+    assert os.path.exists(original_path)
+    assert os.path.exists(preview_path)
+
+
+def test_clean_orphans_should_not_remove_files_associated_with_documents_without_preview(settings):
+    metadata = ('title,summary,credits,path\n'
+                'my video,my video summary,BSF,a-video.mp4\n')
+    write_metadata(metadata)
+    call_command('import_medias', CSV_PATH)
+    original_path = os.path.join(settings.MEDIA_ROOT, 'mediacenter/document', 'a-video.mp4')
+    assert os.path.exists(original_path)
+    call_command('remove_medias', 'clean_orphans')
+    assert os.path.exists(original_path)
+
+
+def test_clean_orphans_should_correctly_remove_ophan_files(settings):
+    metadata = ('title,summary,credits,path,preview\n'
+                'my video,my video summary,BSF,a-video.mp4,an-image.jpg\n')
+    write_metadata(metadata)
+    call_command('import_medias', CSV_PATH)
+    original_path = os.path.join(settings.MEDIA_ROOT, 'mediacenter/document', 'a-video.mp4')
+    preview_path = os.path.join(settings.MEDIA_ROOT, 'mediacenter/preview', 'an-image.jpg')
+    Document.objects.all().delete()
+    assert os.path.exists(original_path)
+    assert os.path.exists(preview_path)
+    call_command('remove_medias', 'clean_orphans')
```

Comment:
Great, now I imagine you working in an orphanage, giving kids their bath.

More seriously though, "orphan" is standard terminology for this kind of things among developers. But it might not be the most obvious things for users, even command line users. Plus it can bring to mind all sorts of unpleasant ideas.

So perhaps that's not the best string to use, at least in user-visible parts?